test/tests/functional/pbs_multi_sched.py

```diff
@@ -544,11 +541,28 @@ def test_job_sorted_per_scheduler(self):
                             {'scheduling': 'True'}, id="sc3")
         self.server.expect(JOB, {'job_state': 'R'}, id=jid4)
 
+    def test_qrun_job(self):
+        """
+        Test jobs are sorted as per job_sort_formula
```

Comment:
qrun doesn't use the job_sort_formula.  It is a cycle where only one job runs.  The job_sort_formula orders the jobs.